src/components/SignIn/index.jsx

```diff
@@ -0,0 +1,128 @@
+import * as firebase from "firebase/app";
+import "firebase/auth";
+import "firebase/firestore";
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import Row from "react-bootstrap/Row";
+import Alert from "react-bootstrap/Alert";
+import "./index.css";
+import { useTranslation } from "react-i18next";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faGoogle } from "@fortawesome/free-brands-svg-icons";
+import { faFacebook } from "@fortawesome/free-brands-svg-icons";
+
+const SignIn = () => {
+  const history = useHistory();
+  const [alert, setAlert] = useState(null);
+
+  var googleProvider = new firebase.auth.GoogleAuthProvider();
```

Comment:
change these to const please

Reply:
> change these to const please

OK. I got them from firebase documentation https://firebase.google.com/docs/reference/js/firebase.auth.GoogleAuthProvider. const may be too strict, I will try const or let accordingly.